### PR TITLE
The points are no longer time based, also modified the readme

### PR DESCRIPTION
**Pull request by Coder-Srinivas**
I have documented my changes over [here](https://docs.google.com/document/d/1Xv9EyliKnnmnZ9In85wlhV-jmt9MDpxrjuiF5wQxDZ4/edit?usp=sharing). Please let me know if any changes are required.

### CODE REVIEW

**Comment by chrisbendel** [comment]
Great job @Coder-Srinivas, looks good so far. There are a couple references to the `reward.startAt` and `reward.endAt` in `rewards.tsx` component. Thats in the admin section when editing/creating rewards if you want to get those too 😃

**Comment by Coder-Srinivas** [comment]
@chrisbendel I will update the PR

File: README.md
Comment (chrisbendel) [inline README.md:1]: Thank you for adding this helpful information! 👍

**Review by chrisbendel — APPROVED** [review]
Great job! Love cleaning out code like this 👍 !

**Comment by Coder-Srinivas** [comment]
Thank you

**Pull request merged**